ui/app/pages/mobile-sync/mobile-sync.component.js

```diff
@@ -47,27 +48,45 @@ export default class MobileSyncPage extends Component {
     }
   }
 
+  handleIdleTimeout () {
```

Comment:
Maybe `startIdleTimeout` would be a better name? "handle" makes me thing this is an event handler or something.